ts/Maps/MapNavigation.ts

```diff
@@ -63,6 +64,7 @@ declare global {
 
         interface MapNavigationChart extends Chart {
             mapNavButtons: Array<SVGElement>;
+            navButtonsGroup: SVGElement;
```

Comment:
Instead of adding a property to the already crowded `Chart` class, add this to the `MapNavigation` class. 

We should do that with `mapNavButtons` too, but that may be out of scope for this PR.